features/arithmetic-expressions-in-transformations.md

```diff
@@ -60,7 +61,7 @@ You can create arithmetic expressions by using arithmetic operators with express
 {% hint style="info" %}
 * If you're using expressions inside a layer, then the layer should be a separate chain transformation.
 
-* Only the `bw`, `bh`, and `bar` expression variables are allowed inside text layers and solid image layers.
+* Only the `bw`, `bh`, and `bar` expression variables are allowed inside text layers, solid image layers and gradient block layers.
```

Comment:
This isn't true? What does `gradient block layers` mean?

Reply:
It's true. Gradient Block Layer means when gradient is used within a canvas layer (just like solid bg used within a canvas layer).

`l-image,i-ik_canvas,e-gradient-sp-bh_div_2,l-end` - Valid
`l-image,i-ik_canvas,e-gradient-sp-ih_div_2,l-end` - Invalid

Reply:
got it

Reply:
resolved